Makefile

```diff
@@ -230,11 +218,6 @@ eunit-fortuna: KIND=test
 eunit-fortuna: internal-build
 	@ERL_FLAGS="-args_file $(EUNIT_VM_ARGS) -config $(EUNIT_SYS_CONFIG) -network_id local_fortuna_testnet" $(REBAR) do eunit $(EUNIT_TEST_FLAGS)
 
-eunit-mnesia-leveled: KIND=test
-eunit-mnesia-leveled: internal-build
```

Comment:
I approve the removal of this target.
I am not convinced re the rest so not approving for now.